### DISCUSSION TITLE
Error Inputs/targets with missing values detected, while there is no missing value

### DISCUSSION THREAD

**Discussion by wtwilley17** [ourownstory/neural_prophet#1521] [Q&A - get help using NeuralProphet]
I got this error =  'Inputs/targets with missing values detected. Please either adjust imputation parameters, or set 'drop_missing' to True to drop those samples.'

Check with 
```
df_train.isnull().sum()
```
and

```
ds    0
y     0
dtype: int64
```

I tried using drop_missing = True, but it ruined my prediction into NA for all values.

my code :
```
m = NeuralProphet( 
    changepoints_range=0.99,
    normalize='standardize', 
    n_changepoints=50,
    trend_reg=0,
    yearly_seasonality=True,
    weekly_seasonality=20,
    daily_seasonality=10,
    quantiles=[0.90, 0.10],
    n_lags = 10,
    n_forecasts = 10
)
m.set_plotting_backend("plotly")
metrics = m.fit(df_train, freq="H")
```

**Comment by mmangione** [comment]
This happens when you have gaps of more than 30 periods in your data. The system cannot impute data for gaps longer than 30 periods, so you must do that yourself, then feed your data frame into the model.
